Felienne/hedy
# Fix #1655

**Description**
Fixed an issue where the 404 page within the quiz `iframe` would result in a redirect to the main page within the iframe. We added an additional argument to the rendering of the 404 page which can be set to through to make sure the template knows to render the `parent` window to the homepage instead of itself.

**Fix for**
This PR fixed #1655.

**How to test**
Try to start a quiz on a non-supported language. When the error button appears: press it and notice that the main page reload correctly instead of the iframe itself.
